payments-model/src/main/java/com/stripe/android/model/CvcCheck.kt

```diff
@@ -9,12 +9,16 @@ enum class CvcCheck(
     Pass("PASS"),
     Fail("FAIL"),
     Unavailable("UNAVAILABLE"),
-    Unchecked("UNCHECKED");
+    Unchecked("UNCHECKED"),
+    Unknown("UNKNOWN");
+
+    val requiresRecollection: Boolean
+        get() = this in setOf(Fail, Unavailable, Unchecked)
```

Comment:
Decided to add an `Unknown` case. Previously, I was defaulting to `Unavailable`, which iOS treats as requiring recollection (see [here](https://github.com/stripe-ios/stripe-ios/blob/cb58f1ee8b37c547df93ce3a6dc58537fd393840/Stripe/PaymentDetails.swift#L225)).